src/mtool/api/Dockerfile-flask

```diff
@@ -32,11 +34,11 @@ COPY ./requirements.txt .
 #RUN sudo apt-get install python3-pip
 #RUN pip3 install --upgrade setuptools
 RUN apt-get update && apt-get -y install sudo 
-RUN pip3 install --upgrade pip
-RUN pip3 install -r ./requirements.txt 
-RUN pip3 install uwsgi -t ./
+RUN pip3 install --trusted-host=files.pythonhosted.org --trusted-host=pypi.org -upgrade pip
```

Comment:
If you make the contents below into a file and copy it from Dockerfile, you don't have to use trusted host every time.

`[global]
proxy = http://12.26.204.100:8080/
cert = /etc/McAfee_Certificate.crt
trusted-host = pypi.python.org pypi.org files.pythonhosted.org pip.pypa.io`

Reply:
good opinion. I will reflect it right away.